opentelemetry/CHANGELOG.md

```diff
@@ -12,6 +12,17 @@
 - Updated the return types of `InstrumentProvider` trait methods to return the instrument instead of a `Result`. [#2227](https://github.com/open-telemetry/opentelemetry-rust/pull/2227)
 - **Breaking change for exporter authors:** Marked `KeyValue` related structs and enums as `non_exhaustive`. [#2228](https://github.com/open-telemetry/opentelemetry-rust/pull/2228)
 - **Breaking change for log exporter authors:** Marked `AnyValue` enum as `non_exhaustive`. [#2230](https://github.com/open-telemetry/opentelemetry-rust/pull/2230)
+- **Breaking change for Metrics users:** The `init` method used to create instruments has been renamed to `build`.
```

Comment:
the changelog about try_init still refers to "init". lets change that also.

Alternatively, we can make a single changelog entry consolidating the user-facing changes. Something like
Removed `try_init` method from instrument builders. To build instruments, user the `build()` method. 
This method always return an instrument - in case of no sdk being enabled or instrument validation failed (invalid name etc.), or the instrument is disabled using views, the instrument internally uses a no-op.
eg:
before:
let counter = meter.u64_counter("my_counter").init();
let counter = meter.u64_counter("my_counter").try_init();
after:
let counter = meter.u64_counter("my_counter").build();